.github/workflows/ci.yaml

```diff
@@ -87,6 +87,23 @@ jobs:
         run: |
           bash ./ci/ensure_formated.sh
 
+  bazel:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          ssh-key: ${{ secrets.SSH_KEY }}
+
+      - name: Run Bazel
+        env:
+          CC: clang-14
```

Comment:
Let's build with the same compiler that the cmake build is using (assuming thats not clang-14).

Reply:
I changed it to gcc-6.